projects/packages/sync/changelog/update-hooks-prevent-doing-it-wrong-notice

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Hooks: Change several action priorities to ensure translation loading is not triggered too early.
```

Comment:
This seems like it could use updating now.

Reply:
Updated it at 38f420cc60b63ea58a883312f78e16be2529641d - I think this is a little better?

Reply:
Looks good to me, thanks!